.gitignore

```diff
@@ -1,5 +1,20 @@
+.idea
+vendor/
+
+# composer
 composer.lock
 composer.phar
-vendor/
+
+# osx
 .DS_Store
-.idea
+.DS_Store?
+._*
+.Spotlight-V100
+.Trashes
+
+# npm
+.npm/node_modules
+node_modules
+
+# generated files
+templates/theme/css
```

Comment:
On the scaffold repo you would never do a npm install or generate the css files, hence I am wondering why the code is in here?